### PR TITLE
Feature/6 pop up time tracking

### PR DESCRIPTION
**Pull request by bryanakitchen**
Created initial bootstrap from yeoman and set up initial pop up and time tracking capabilities. This will allow us to test publishing a VSCode extension.

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+	if(response === 'Yes') {
+		vscode.window.showInformationMessage('Have a great day!');
+		
+		setInterval(() => {
+			const lapTime = new Date();
+
+			const incrementOfTime = Math.round((lapTime - startTime) / 60000);
+			
+
+			vscode.window.showInformationMessage(`You have been working for ${incrementOfTime} minutes.`, 'Disable');
```
Comment (RhoneLachner) [inline extension.js:25]: Remove Disable button :-)

**Review by RhoneLachner — APPROVED** [review]
Looking goooooood

File: extension.js (hunk 1 above)
Comment (jillianlg) [reply extension.js:25]: Good catch!

**Review by jillianlg — APPROVED** [review]
Nice job team!

**Pull request merged**